### PR TITLE
Issue 67

### PR DESCRIPTION
**Pull request by silenius**
This is a fix for #67 with changes from #66 included

Close #66.

### CODE REVIEW

File: src/zope/sqlalchemy/datamanager.py [hunk 1]
```diff
@@ -84,22 +85,18 @@ def __init__(
...
-
-        # Support both SQLAlchemy 1.0 and 1.1
-        # https://github.com/zopefoundation/zope.sqlalchemy/issues/15
-        _iterate_parents = (
-            getattr(transaction, "_iterate_self_and_parents", None)
-            or transaction._iterate_parents
-        )
-
-        self.tx = _iterate_parents()[-1]
+        if sqlalchemy_version >= "1.4.0":
```
Comment (mgedmin) [inline src/zope/sqlalchemy/datamanager.py:88]: This is not the right way to compare versions.  Consider, for example, sqlalchemy_version of `"1.10.0"`.

**Comment by icemac** [comment]
All tests are breaking on GHA. On `master` at least Python 3.6+ and lint are successful. Could you please have a look?

**Comment by silenius** [comment]
it should be OK for >3.5 (tests are broken for < 3.6 (as you noticed in #69))

**Comment by icemac** [comment]
@silenius The `lint` checks still fail. For Python < 3.6 I am preparing a fix at #70.

**Comment by icemac** [comment]
#70 is now merged, I added the changes to this PR, too.

**Comment by icemac** [comment]
Thank you for your contribution.

According to the contributing policies of the zopefoundation organization you need to sign a contributor agreement before any non-trivial change can be merged. For details please consult the [Contributing guidelines for zopefoundation projects](https://github.com/zopefoundation/.github/blob/master/CONTRIBUTING.md).

**Comment by silenius** [comment]
Thank you for looking at this. I'll check the contributor agreement document as soon as I have a little time and keep you updated..

**Comment by silenius** [comment]
I've completed and sent the PDF document to agreements@plone.org

**Review by icemac — APPROVED** [review]
LGTM.

In `zopefoundation` repositories the policy is that the creator of the PR merges it, to prove having signed the contributor agreement.

**Comment by silenius** [comment]
:heavy_check_mark:

**Comment by icemac** [comment]
I am going to write a change log entry and to cut a release now.

**Comment by icemac** [comment]
I just released https://pypi.org/project/zope.sqlalchemy/1.5/

**Pull request merged**